draco/asp/helpers.lp

```diff
@@ -4,3 +4,21 @@
 attribute(mark_channel,M,C) :-
     entity(encoding,M,E),
     attribute((encoding,channel),E,C).
+
+% @helper(mark_scale_channel) Whether a mark has a scale for a channel.
+attribute(mark_scale_channel,M,S,C) :-
```

Comment:
This can probably be `attribute((mark, scale_type),M,T)` since we don't need the full scale in marks right now. What is the reason that you decided for `mark_scale_channel`?